tests/test_plugin.py

```diff
@@ -26,6 +26,18 @@ def check_album(actual, expected):
     assert vars(actual) == vars(expected)
 
 
+@pytest.mark.parametrize(
+    "comments, expected_url",
+    [
+        ("Visit https://label.bandcamp.com", "https://label.bandcamp.com"),
+        ("Visit https://supercommuter.net", "https://supercommuter.net"),
+        ("Visit https://no-top-level-domain", None),
```

Comment:
Hey, sorry I'm a little late to this, but is it worth adding a test case for a URL with more than two domain parts, e.g. a sub-domain of a country TLD like `https://myband.com.au/` or `https://myband.co.uk/`? I'm not sure if your regular expression handles more than one dot in the domain portion.

Reply:
To be honest I did not even give a chance for anyone to have a look at this 😅 

This is a good idea, of course! Would you like to contribute yourself by any chance?

Reply:
I'll see if I can find time over the next couple days. Otherwise, feel free to give it a go yourself.

Reply:
Also, I am in Australia, so I think we're battling timezones a little. 😅

Reply:
I checked the URLs that you provided and it seems to handle them well, but I'm excited to see whether you could find a URL that breaks it!